### ISSUE TITLE
Saturation makes the screen black and white when lower than -50%

### ISSUE THREAD

**Issue by Turbolqk** [whatawurst/android_device_sony_yoshino-common#105]

**Comment by derfelot** [comment]
Yeah, this bug has been around for a long time. But I have no idea how to fix it. All I can say that two other devices of mine have the exact same issue, and they are official LOS devices. So seems to be a rather common issue.
